TOOLS/odoo-tools.sh

```diff
@@ -1200,20 +1210,18 @@ if [ "$SCRIPT_MODE" = "updatetranslation" ]; then
 
     # ------------------------------------------ ADMIN POST PREPARATION 8 - 9 - 10 BEGIN -----------------------------------------
 
-    echo "Temporary backup of Database ${DBNAME} after udpate..."
-    ${INSTANCE_PATH}/TOOLS/db-tools.py -b ${BASEPORT69} -s "${SUPER_PASSWORD}" backup -d ${DBNAME} -f ${TMPFILENAME}_after.zip
-    # TODO: change this recursive call odoo_tools.sh later or use only pgdump for this temp backup
-    #sudo - postgres ${DBNAME} > ${TMPFILENAME}_before.sql
     echo "Starting up Database ...."
     service ${DBNAME} start
+    sleep 10
```

Comment:
Avoid sleeps - bad practice - check if the service is up and running by "service Status" or by grabbing the log of the database or with ps or with a wget check :)